### ISSUE TITLE
Cannot paste text into alt text field of an image block--seems to be intermittent

### ISSUE THREAD

**Issue by vr619536** [inclusive-design/idrc#787]
## Describe the bug

On a project page, when an image is added, the paste function does not work in the alt text field. Both Caren (mac) and I (PC firefox and pretty sure chrome but today it worked) observed the same issue. I sometimes find that I am able to paste text in this field if there is already text there. After I reproduced the problem, I tried to reproduce it again and I wasn't able to  so I guess it's intermittent. I had assumed it was my computer until I was on a call with Caren and she experienced the same issue.
## To Reproduce

1.  from the collections in the CMS select "projects"
2.  select Regulating the digital Domain
3.  English language interface: in the body (optional) section find the image block with the project logo.
5.  delete the alt text (Regulating the Digital Domain logo) so that the field is empty
6.  copy text from anywhere
7. try to paste your copied text into the alt text field however you like--menus, keyboard shortcut

## Expected behavior
copied text should appear when pasted.

## Screenshots

![image](https://github.com/inclusive-design/idrc/assets/145715822/8359c5e9-7ced-43cd-bd36-c2fedf7f1adf)

## Desktop (please complete the following information):

PC Windows 11 Pro
Firefox 119 that doesn't seem to want to update despite my best efforts

Mac--going to have to ask Caren or check one of her bug reports.

## Smartphone (please complete the following information):

N/A
